README.md

```diff
@@ -1,2 +1,174 @@
+## The Catalog - My Things
+
+<a name="readme-top"></a>
+<div align="center">
+    <img src="logo_jcm.png" alt="main-logo" width="500"  height="auto" />
+  <br/>
+  <h3><b>The Catalog - My Things</b></h3>
+</div>
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 The Catalog - My Things ](#-the-catalog---my-things-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Presentation ](#-presentation-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+# 📖 The Catalog - My Things <a name="about-project"></a>
+
+The Catalog - My Things is an app developed to create a catalog of personal things, music, books, and games. This app is developed using Ruby, Object-Oriented Programming (OOP) principles, and unit testing using RSpec. This is a console user interface.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://ruby-doc.org/">Ruby</a></li>
+  </ul>
+
+### Key Features <a name="key-features"></a>
+- **Create necessary associations within the project to lay the foundation for the user interface development planned for the next milestone.**
+- **Add a console User Interface to run the application.**
+- **Refactor the entire application to apply SOLID, YAGNI, KISS, & DRY principles.**
+- **Preserve all data using files with JSON format.**
+- **Use of Object Oriented Programming Principles.**
+- **Add unit testing with RSpec.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+ <!-- LIVE DEMO -->
+
+## 🚀 Presentation <a name="presentation"></a>
+
+- Click [here](https://youtu.be/9LO_0zhpoC4).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+[Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
+
+### Setup
+
+```sh
+  cd my-folder
+  git clone https://github.com/jcmunav63/catalog-my-things.git
+```
+
+### Install
+
+This project does not require additional dependencies, just Ruby interpreter. However, it uses RuboCop as a linter and you should set it up as well.
+
+```sh
+bundle install
+```
+
+### Usage
+
+To run the project, navigate to the project directory and execute the following command:
+
+```sh
+  ruby main.rb
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+Run the following command
+```sh
+rubocop
```

Comment:
- [ ] You did an amazing job guys! 👌 Let's, however, include appropriate instructions on how to run test cases, using the command `rspec` instead of `rubocop`

Reply:
Thank you very much for your time en feedback.

Reply:
Fixed.